src/test/java/testresults/annual_certificates/TestPostTestResultsAnnualCertificateGenerationHgv.java

```diff
@@ -277,6 +278,13 @@ public void testResults_Annual_Certificate_Generation_Hgv() {
 
         // Post the results, together with any alterations, and verify that they are accepted.
         testResultsSteps.postVehicleTestResultsWithAlterations(testResultRecord, alterations);
+        if ("warmup test".equals(name)) {
```

Comment:
as per Craig's query, does this throw an exception if the status response is not 201 or is it just an assertion fail? We cant run this locally so if this is working as you expect, please disregard these comments

Reply:
Ran it locally and the tests, pass, added a sout to print out a Retry and exception if it returns anything other than 201

Reply:
Added sout on all 3 classes.